openmdao/core/mpi_wrap.py

```diff
@@ -53,6 +53,7 @@ def under_mpirun():
         if name == 'OMPI_COMM_WORLD_RANK' or \
            name == 'MPIEXEC_HOSTNAME' or \
            name.startswith('MPIR_') or \
+           (name.startswith('MSMPI_') and name.startswith('PMI_RANK')) or \
```

Comment:
Turns out that MSMPI which is based on mpich sets MPIEXEC_HOSTNAME, so i'll go ahead and remove that line that i added